### PR TITLE
feature/globalaction-field-save-yaml-specified

### PR DESCRIPTION
**Pull request by RickardAhlstedt**
* References [#15446](https://github.com/pimcore/pimcore/pull/15446) and [15447](https://github.com/pimcore/pimcore/pull/15447)
* Added variable for fetching the option `saveSubject` from the currently submitted globalaction-event and passing it to WorkflowManager.

### CODE REVIEW

**Comment by RickardAhlstedt** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by RickardAhlstedt** [comment]
recheck

File: src/Controller/Admin/WorkflowController.php [hunk 1]
```diff
@@ -166,8 +166,10 @@ public function submitGlobalAction(Request $request, Registry $workflowRegistry,
         $workflowOptions = $request->get('workflow', []);
         $workflow = $workflowRegistry->get($this->element, $request->get('workflowName'));
 
+        $saveSubject = $workflowManager->getGlobalAction( 'product_status_workflow', $request->get('transition') )->getSaveSubject() ?? true;
```
Comment (fashxp) [inline src/Controller/Admin/WorkflowController.php]: hardcoded name of a workflow? that can't be right here.

**Review by fashxp — CHANGES_REQUESTED** [review]
we also bump requirement of pimcore/pimcore to ^11.1 in composer.json

as long as pimcore:pimcore:11.1.0 is not released, we eventually need to add 
```
  "prefer-stable": true,
  "minimum-stability": "dev"
```
to composer.json to make phpstan happy

File: src/Controller/Admin/WorkflowController.php (hunk 1 above)
Comment (RickardAhlstedt) [reply src/Controller/Admin/WorkflowController.php]: Sorry, that slipped by when I was pulling this from my project into this PR, I will change it and update the PR

**Comment by RickardAhlstedt** [comment]
@fashxp Updated the hardcoded variable-name, as well as updated the dependency for pimcore from ^11.0 to ^11.1 as mentioned above.

**Review by fashxp — APPROVED** [review]

**Comment by fashxp** [comment]
looks good now, we'll merge after 1.1.0 is released ... as we want to have one more release that is compatible to pimcore/pimcore:11.0

**Comment by robertSt7** [comment]
@RickardAhlstedt Could you please resolve the conflicts? Thanks

**Comment by RickardAhlstedt** [comment]
@robertSt7 The conflict has been resolved 👍

**Pull request merged**